environments.json

```diff
@@ -1595,50 +1595,117 @@
+    "platformtype": "mobile",
+    "release": "2009-04-30",
+    "obsolete": "very"
+  },
+  "android1_6": {
+    "full": "Android Browser 1.6 (Donut)",
+    "family": "Android Browser",
+    "short": "AN 1.6",
+    "platformtype": "mobile",
+    "release": "2009-09-15",
+    "obsolete": "very"
+  },
+  "android2_0": {
+    "full": "Android Browser 2.0 (Eclair)",
+    "family": "Android Browser",
+    "short": "AN 2.0",
+    "platformtype": "mobile",
+    "release": "2009-10-26",
+    "obsolete": "very"
+  },
+  "android2_1": {
+    "full": "Android Browser 2.1 (Eclair)",
+    "family": "Android Browser",
+    "short": "AN 2.1",
+    "platformtype": "mobile",
+    "release": "2010-01-12",
+    "obsolete": "very"
+  },
+  "android2_2": {
+    "full": "Android Browser 2.2 (Froyo)",
+    "family": "Android Browser",
+    "short": "AN 2.2",
+    "platformtype": "mobile",
+    "release": "2010-05-20",
+    "obsolete": "very"
+  },
+  "android2_3": {
+    "full": "Android Browser 2.3 (Gingerbread)",
+    "family": "Android Browser",
+    "short": "AN 2.3",
+    "platformtype": "mobile",
+    "release": "2010-12-06",
+    "obsolete": "very"
+  },
+  "android3_0": {
+    "full": "Android Browser 3.0 (Honeycomb)",
+    "family": "Android Browser",
+    "short": "AN 3.0",
+    "platformtype": "mobile",
+    "release": "2011-02-22",
+    "obsolete": "very"
+  },
+  "android3_1": {
+    "full": "Android Browser 3.1 (Honeycomb)",
+    "family": "Android Browser",
+    "short": "AN 3.1",
+    "platformtype": "mobile",
+    "release": "2011-05-10",
+    "obsolete": "very"
+  },
+  "android3_2": {
+    "full": "Android Browser 3.2 (Honeycomb)",
+    "family": "Android Browser",
+    "short": "AN 3.2",
+    "platformtype": "mobile",
+    "release": "2011-07-15",
+    "obsolete": "very"
+  },
   "android4_0": {
-    "full": "Android Browser",
+    "full": "Android Browser 4.0 (Ice Cream Sandwich)",
+    "family": "Android Browser",
     "short": "AN 4.0",
     "platformtype": "mobile",
-    "obsolete": true
+    "release": "2011-10-18",
+    "obsolete": "very"
   },
   "android4_1": {
-    "full": "Android Browser",
+    "full": "Android Browser 4.1 (Jellybean)",
+    "family": "Android Browser",
     "short": "AN 4.1",
     "platformtype": "mobile",
-    "obsolete": true
+    "release": "2012-07-09",
+    "obsolete": "very"
   },
   "android4_2": {
-    "full": "Android Browser",
+    "full": "Android Browser 4.2 (Jellybean)",
+    "family": "Android Browser",
     "short": "AN 4.2",
     "platformtype": "mobile",
-    "obsolete": true
+    "release": "2012-11-13",
+    "obsolete": "very"
   },
   "android4_3": {
-    "full": "Android Browser",
+    "full": "Android Browser 4.3 (Jellybean)",
+    "family": "Android Browser",
     "short": "AN 4.3",
     "platformtype": "mobile",
-    "obsolete": true
+    "release": "2013-07-24",
+    "obsolete": "very"
   },
   "android4_4": {
-    "full": "Android Browser",
+    "full": "Android Browser 4.4 (KitKat)",
```

Comment:
I don't think this is correct. Android Browser ended with Android 4.3.

Also, it lacks  the `"equals": "chrome30"` declaration which was previously present. Doesn't this break correctness of the results?

As for me, Android 4.4 & newer was never correct as there's no Android Browser in Android 4.4 & newer so I'd be fine with removing them (If maintainers agree).